composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "phpoffice/phpexcel",
+    "name": "forka-package/phpexcel",
```

Comment:
I'm sorry. I forget to create an independent branch. :)
Thanks a lot. I'll do it and I'll update this PR.